packages/webdriverio/src/commands/network/throttle.js

```diff
@@ -1,4 +1,8 @@
-/* istanbul ignore file */
+/**
+ * some description
```

Comment:
are you going to add description?

Reply:
Yes, it is part of this PR: https://github.com/webdriverio/webdriverio/pull/5472/files#diff-0a3a23f19504d4292082bcf209d07d31R1-R26